lib/signaling/v2/room.js

```diff
@@ -372,4 +383,20 @@ function periodicallyPublishStats(roomV2, localParticipant, transport, intervalM
   });
 }
 
+function handleSubscriptionFailures(room) {
+  var remoteTracks = Array.from(room.participants.values()).reduce(function(remoteTracks, participant) {
```

Comment:
@markandrus This can be re-written as:
```javascript
var remoteTracks = new Map(flatMap(room.participants, participant => {
  return flatMap(participant.tracks, track => [track.sid, track]);
}));
```
?

Reply:
good idea 👍